.travis.yml

```diff
@@ -64,7 +64,7 @@ jobs:
     script:
     - set -e
     - $TRAVIS_BUILD_DIR/scripts/travis/build.sh
-    - scripts/test/setup_k8s_env.sh
+    - $TRAVIS_BUILD_DIR/scripts/travis/setup_minikube.sh
```

Comment:
I moved `test/setup_k8s_env.sh` to `travis/setup_minikube.sh` because this script is specifically tailored for running in Travis VMs.  For example, it contains a line

```bash
sudo chown -R travis: $HOME/.minikube/
```

In some successive PRs, we can remove such dependencies and make it reusable for SQLFlow Desktop deployment.